extensions/Vendor/AVR_texture_offset_tile/schema/AVR_texture_offset_tile.texture.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema" : "http://json-schema.org/draft-04/schema",
+    "title" : "AVR_texture_offset_tile texture extension",
+    "type" : "object",
+    "properties" : {
+        "offsetS": {
+            "type": "number",
+            "description": "How much a single repetition of the texture is offset from the beginning, in the S direction.",
+            "minimum": -1.0,
+            "maximum": 1.0,
+            "default": 0.0
+        },
+        "offsetT": {
+            "type": "number",
+            "description": "How much a single repetition of the texture is offset from the beginning, in the T direction.",
+            "minimum": -1.0,
+            "maximum": 1.0,
+            "default": 0.0
+        },
+        "tileS": {
+            "type": "number",
+            "description": "How many times the texture is repeated across the surface, in the S direction.",
```

Comment:
Since `tile` is `[-1, 1]`, could description be rephrased to not use "How many times"?
Text from readme (size of region) seems to be more precise, imho.